docs/language/accounts.mdx

```diff
@@ -49,10 +70,9 @@ Every account can be accessed through two types:
 
       let contracts: AuthAccount.Contracts
 
-      // Key management
+      // Keys
 
-      fun addPublicKey(_ publicKey: [UInt8])
-      fun removePublicKey(_ index: Int)
```

Comment:
Maybe leave these functions document, but indicate that they are now deprecated and the new account key API should be used. Maybe just copy https://github.com/onflow/cadence/pull/635/files#diff-922d8e6ca000c087b0eba7a1886b7bccde23a0457bf3412dda16121293a372aeR194 here 👌